libkbfs/block_retrieval_queue.go

```diff
@@ -179,8 +179,8 @@ func (brq *blockRetrievalQueue) Request(ctx context.Context, priority int, kmd K
 				brq.config.BlockCache().GetWithPrefetch(ptr)
 			if err == nil && cachedBlock != nil {
 				block.Set(cachedBlock, brq.config.codec())
-				brq.triggerPrefetchAfterBlockRetrieved(
-					cachedBlock, kmd, priority, hasPrefetched)
+				brq.Prefetcher().PrefetchAfterBlockRetrieved(
```

Comment:
I don't think this can be a `go` invocation, because we need the cache to have the entry when the channel returns, otherwise we could potentially race against future cache `Get`s.